src/components/InputField/index.js

```diff
@@ -1,18 +1,24 @@
 import React from 'react';
 import './InputField.css';
 
-const InputField = ({ label, id, name }) => (
-  <div className="formRow">
-    <label htmlFor={id}>{label}</label>
-    <input
-      style={{ marginLeft: '4px' }}
-      type="text"
-      name={name}
-      id={id}
-      className="formInput"
-      ref={input => (this.tooltip = input)}
-    />
-  </div>
-);
+const InputField = ({ label, id, name, onChange }) => {
 
-export default InputField;
+  const wrappedOnChange = (event) => onChange(event.target.value);
+
+  return (
+    <div className="formRow">
+      <label htmlFor={id}>{label}</label>
+      <input
+        style={{ marginLeft: '4px' }}
+        type="text"
+        name={name}
+        id={id}
+        className="formInput"
+        onChange={wrappedOnChange}
+        ref={input => (this.name = input)}
```

Comment:
We don't need this anymore since we are using `onChange` to get the field value, instead of DOM API which we use via `ref`.